.github/RELEASE_CHECKLIST.md

```diff
@@ -3,7 +3,7 @@
 
 ## Code
 - [] Ensure docs exist at singularityware.github.io for new/updated features
-- [] Update version number in `configure.ac`
+- [] Update version number in `configure.ac` and `INSTALL.md`
```

Comment:
Is this supposed to be here?  Does it conflict with https://github.com/singularityware/singularity/pull/1112?

Reply:
They are different things - that PR adds it to the debian install files, and this one is general configuration of Singularity.

Reply:
OK thanks.